Example/Tests/Classes/ActionManagerTest.m

```diff
@@ -327,4 +308,20 @@ - (void)test_active_period_true
     XCTAssertTrue(result.matchedActivePeriod);
 
 }
+
+#pragma mark Helpers
+
+-(NSDictionary *)_configForActivePeriodMatch:(BOOL)activePeriod
+{
+    NSDictionary *config = @{@"whenLimits":@{@"children":@[]},
+                             @"whenTriggers":@{@"children":@[@{@"noun":@"ActivePeriodTest",
```

Comment:
completely a side note. If we have a parameter that takes a trigger as a parameter we can reuse this method in test_matched_trigger as well. Not that important though.